test/src/test/java/org/corfudb/CorfuTestParameters.java

```diff
@@ -90,6 +96,7 @@ public CorfuTestParameters(){
         TIMEOUT_SHORT = Duration.of(1, SECONDS);
         TIMEOUT_NORMAL = Duration.of(10, SECONDS);
         TIMEOUT_LONG = Duration.of(2, MINUTES);
+        TIMEOUT_VERY_LONG = Duration.of(4, MINUTES);
```

Comment:
On github actions longer timeouts make it harder for failure cases to fail quickly and in the end it hurts our end-to-end runs by making things longer. Can we please consider an alternative than increasing test timeout to 4 minutes for reliabilty?

Reply:
Currently only 2 test cases were added specifically for this case. They couldn't finish within 2mins locally. I see your point but didn't find an alternative approach to address it. Please let me know if there is any